Update readme (#1)
DevExpress-Examples/asp-net-web-forms-popup-use-one-popup-for-entire-application: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T501713)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Popup Control for ASP.NET Web Forms - How to use one pop-up across all pages of an application

You can use one [Popup Control](https://docs.devexpress.com/AspNet/3582/components/docking-and-popups/popup-control) across all pages of your application to improve performance. This example demonstrates how to create a popup control on the master page and invoke it from multiple pages. The pop-up window's content depends on the page that shows this window.

![Popup Window](popup.png)

> In this example, the **WebForm1** page populates the pop-up window with content during control initialization. The **WebForm2** page specifies the window's content once the control sends a callback to the server. Use **Init approach** and **Callback approach** menu items to switch between these pages. Click the **Show Popup** button to show the pop-up window.

Reviewer: We get used to phrases like "the Height property" or "the Button class", but sometimes it's more natural to mention the name after the noun. I'm not sure why exactly it feels more natural to switch the order in this case. Maybe it's because of the numbers in the names of pages. In the third sentence, you need to read through two long names before you understand what the sentence talks about (menu items). 

 ```suggestion
In this example, the page **WebForm1** populates the pop-up window with content during control initialization. The page **WebForm2** specifies the window's content once the control sends a callback to the server. Use menu items **Init approach** and **Callback approach** to switch between these pages. Click the **Show Popup** button to show the pop-up window.
```

## Overview

Follow the steps below to use one pop-up across multiple pages:

1. Place an empty popup control in the root master page.

> 2. Handle one of the following events:

Reviewer: I think it's too early to say "handle". For me, the word "handle" implies that we add code that handles the event. So if you start a list item with the words "Handle an event", I assume the entire "event handling" will be covered in this item. But here we only specify the attribute. 

 ```suggestion
2. Select the event that will generate pop-up window content:
```

    * The `Init` event allows you to create pop-up window content at runtime during popup control initialization.
    * The [WindowCallback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControlBase.WindowCallback) event allows you to dynamically create pop-up window content once the window sends a callback.

>     The code snippet below handles the `Init` event:

Reviewer: ```suggestion
    The code snippet below assigns a handler to the `Init` event:
```

    ```aspx
    <dx:ASPxPopupControl ID="popupControl" runat="server" ClientInstanceName="popup" 
        OnInit="popupControl_Init" />
    ```

> 3. Create a delegate for the event you handled on the previous step:

Reviewer: I'm just continuing with the same edit. The reader hasn't handled the event yet - they keep adding code that will eventually handle it. 

```suggestion
3. Create a delegate for the event you selected in the previous step:
```
